EIPS/eip-5187.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 5187
+title: Extend EIP-1155 with rentable usage rights
+description: Separate ownership and usage rights of EIP-1155 to allow users to use NFTs for an allotted time and return them to owners after expiration.
+author: DerivStudio (@DerivStudio)
+discussions-to: https://ethereum-magicians.org/t/eip-draft-extending-erc1155-with-rentable-usage-rights/9553/4
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-17
+requires: 721, 1155
```

Comment:
I'd suggest adding EIP-165 for better detection

Reply:
Thanks for your suggestion.EIP-165 has been added.